tests/_test_msui/test_mscolab.py

```diff
@@ -585,13 +572,13 @@ def test_update_category(self, mocktext):
         assert self.window.mscolab.active_operation_category == "new_category"
 
     @mock.patch("PyQt5.QtWidgets.QMessageBox.information")
-    def test_any_special_category(self, mockbox):
+    def test_any_special_category(self, mockbox, qtbot):
         self._connect_to_mscolab()
         modify_config_file({"MSS_auth": {self.url: "something@something.org"}})
-        self._create_user("something", "something@something.org", "something")
-        self._create_operation("flight1234", "Description flight1234")
+        self._create_user(qtbot, "something", "something@something.org", "something")
+        self._create_operation_successfully(qtbot, "flight1234", "Description flight1234")
         QtTest.QTest.qWait(0)
-        self._create_operation("flight5678", "Description flight5678", category="furtherexample")
+        self._create_operation_successfully(qtbot, "flight5678", "Description flight5678", category="furtherexample")
```

Comment:
not sure if I understand the reason of the postfix "successfully". can we be unsuccessfull too?

Reply:
The `_create_operation_successfully` method ensures, through additional asserts and waiting, that the operation is successfully created and visible in the UI, as opposed to the plain `_create_operation`, which does not have these checks. I.e. due to the asynchronous nature of Qt it is possible that after a `_create_operation` the operation is not yet visible in the operations list while after a `_create_operation_successfully` it will be because it waits for it (or it will fail the test if creating the operation failed).

That is why I chose this suffix. Do you have a better name for this method?

Reply:
There are three uses of `_create_operation` left which try to create invalid operations and assert that a corresponding error is emitted. They are here:
https://github.com/Open-MSS/MSS/blob/9a9345c50edf50bace6f75c359d54e4071d0f7da/tests/_test_msui/test_mscolab.py#L442-L452

I think we could rename `_create_operation_successfully` back into `_create_operation` and name this other one `_create_operation_unchecked` or something like that, to have a shorter name for the more common operation (and be more inline with `_create_user`, which does similar checks but doesn't have an additional suffix since there is just one version of it).